multiqc/modules/nonpareil/nonpareil.py

```diff
@@ -0,0 +1,295 @@
+                "description": f"Total raw sequences ({config.read_count_desc})",
+                "modify": lambda x: x * config.read_count_multiplier,
+                "min": 0,
+                "scale": "GnBu",
+                "format": "{:,.2f} " + config.read_count_prefix,
+                "shared_key": "read_count",
+                "hidden": True,
+            },
+            "LR": {
+                "title": f"{config.base_count_prefix} Sequencing effort",
+                "description": f"Total base pairs sequenced ({config.base_count_desc})",
+                "modify": lambda x: x * config.base_count_multiplier,
+                "min": 0,
+                "scale": "Blues",
+                "format": "{:,.2f} " + config.base_count_prefix,
+                "shared_key": "base_count",
+                "hidden": True,
+            },
+            "overlap": {
+                "title": "Read overlap",
+                "description": "Minimum read overlap",
+                "min": 0,
+                "suffix": " bps",
+                "scale": "Blues",
+                "hidden": True,
+                "format": "{:,.0f}",
+            },
+            "log.sample": {
+                "title": "Log sampling",
+                "description": "Multiplier of the log-sampling (or zero if linear)",
+                "max": 1,
+                "min": 0,
+                "scale": "BuGn",
+                "hidden": True,
+                "format": "{:,.2f}",
+            },
+            "kappa": {
+                "title": "Redundancy",
+                "description": "Dataset redundancy",
+                "modify": lambda x: x * 100,
+                "max": 100,
+                "min": 0,
+                "suffix": "%",
+                "scale": "RdYlGn-rev",
+                "format": "{:,.2f}",
+            },
+            "C": {
+                "title": "Coverage",
+                "description": "Dataset coverage",
+                "modify": lambda x: x * 100,
+                "max": 100,
+                "min": 0,
+                "suffix": "%",
+                "scale": "RdYlGn",
+                "format": "{:,.2f}",
+            },
+            "consistent": {
+                "title": "Consistent Data?",
+                "description": "Is the data sufficient for accurate estimation?",
+                "modify": lambda x: "Yes" if x == 1 else "No",
+                "scale": False,
+                "hidden": True,
+            },
+            "star": {
+                "title": "Ideal coverage",
+                "description": "Objective coverage in percentage; i.e., coverage value considered near-complete.",
+                "max": 100,
+                "min": 0,
+                "suffix": "%",
+                "scale": False,
+                "hidden": True,
+            },
+            "LRstar": {
+                "title": f"{config.base_count_prefix} Sequencing effort for ideal coverage",
```

Comment:
This title is a bit too long, it skews all of the column width..
```suggestion
                "title": f"{config.base_count_prefix} ideal seq",
```